scripts/ci-validate-testcases.sh

```diff
@@ -5,7 +5,7 @@ make generate-testcases
 
 # check differences
 cd sszgen/testcases
-if [[ `git status --porcelain` ]]; then
+if [[ `git status --porcelain .` ]]; then
```

Comment:
Only check the current (`sszgen/testcases`) directory, otherwise it picks up uncommitted changes in the rest of the project.